src/server/editorServices.ts

```diff
@@ -1197,6 +1201,11 @@ namespace ts.server {
         private forEachConfigFileLocation(info: ScriptInfo,
             action: (configFileName: NormalizedPath, canonicalConfigFilePath: string) => boolean | void,
             projectRootPath?: NormalizedPath) {
+
+            if (this.syntaxOnly) {
+                return undefined;

```

Comment:
I thought the idea was to create program but not bind it. Are we not going to do that. This means if file opened is from configured project, it wont pick its compiler options?